frontend/src/container/TopNav/DateTimeSelectionV2/index.tsx

```diff
@@ -0,0 +1,474 @@
+import './DateTimeSelectionV2.styles.scss';
+
```

Comment:
This file would require a complete re-write. Let's create a tech debt item for the same.